main.py

```diff
@@ -28,6 +28,8 @@
 import time
 import numpy as np
 from cmd_args import parse_args
+import pandas as pd
+import pyBigWig
```

Comment:
once `df_to_bigwig` is moved into its own module this import can be removed too from this file (into the bigwigio file)

Reply:
done.